Core/HLE/sceKernelSemaphore.cpp

```diff
@@ -457,3 +458,29 @@ int sceKernelPollSema(SceUID id, int wantedCount)
 	}
 }
 
+static u32 hleUtilsBufferCopyWithRange(u32 outAddr, int outSize, u32 inAddr, int inSize, int cmd)
+{
+		int temp = sceUtilsBufferCopyWithRange((u8*)outAddr, outSize, (u8*)inAddr, inSize, cmd);
+		INFO_LOG_REPORT_ONCE(hleUtilsBufferCopyWithRange,SCEKERNEL, "sceUtilsBufferCopyWithRange(%08x,%i,%08x,%i,%x) should return %i",outAddr,outSize,inAddr,inSize,cmd,temp);
```

Comment:
No sorry, I mean to log something special if there's an error, like
```
if (temp != 0) {
  ERROR_LOG(SCEKERNEL, "hleUtilsBufferCopyWithRange: Failed with %d", temp);
}
return 0;
```
I don't think we need to use REPORT for this, it's quite obscure.

Also the indentation is still one level too much, look at the diff here.